gandalf.js

```diff
@@ -1,18 +1,18 @@
-const formattedDate = new Date().toLocaleDateString();
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!